src/docs/release/breaking-changes/dialog-border-radius.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Dialogs' Default BorderRadius
+description: The default BorderRadius of Dialog widgets is changing.
+---
+
+## Summary
+
+Instances of `Dialog`, as well as `SimpleDialog`, `AlertDialog`, and `showTimePicker`, now has
+a default shape of a `RoundedRectangleBorder` with a `BorderRadius` of 4.0 pixels. This matches the
+current specifications of Material Design. Prior to this change, the default behavior for
+`Dialog.shape`'s `BorderRadius` was 2.0 pixels.
+
+## Context
+
+Use of `Dialog`s and their associated subclasses (`SimpleDialog`, `AlertDialog`, and
+`showTimePicker`), appears slightly different as the border radius is larger. Golden file tests
+will fail if they contain a Dialog` in the new default behavior.
+
+This change was initially introduced with an opt-in flag
+`useMaterialBorderRadius`, to allow developers time to migrate
+without experiencing broken code. This temporary opt-in flag will be removed in a follow up change.
+
+The `showDatePicker` dialog already matches this specification and is unaffected by this change.
+
+## Migration guide
```

Comment:
sounds like really there's no migration unless you want to keep the old behavior or unless you have goldens; i would rephrase this section to be more in line with those use cases.